src/pymola/tree.py

```diff
@@ -739,7 +738,8 @@ def exitSymbol(self, tree: ast.Symbol):
     #def exitClassModificationArgument(self, tree: ast.ClassModificationArgument):
     #     assert isinstance(tree.value, ast.ElementModification), "Found unhandled redeclaration in instance tree."
 
-    def exitInstanceClass(self, tree: ast.InstanceClass):
+    @staticmethod
+    def exitInstanceClass(tree: ast.InstanceClass):
```

Comment:
This does not look right. It should have the same signature as all the other methods. If PyCharm suggests this, there's probably something wrong somewhere else?